spec/core/ics-002-client-semantics/README.md

```diff
@@ -315,6 +315,18 @@ type verifyClientConsensusState = (
   => boolean
 ```
 
+`verifyClientState` verifies a proof of the client state of the specified client stored on the target machine.
+
+```typescript
+type verifyClientState = (
```

Comment:
Rather than adding this function to the specification, we should transition to using the generic verify membership methods. Though I suppose that can be done in a separate PR

Reply:
If you would like to open an issue describing specifically what's to be done, I'd be happy to take a stab at it in another PR